deploy/kubernetes/hcloud-csi-master.yml

```diff
@@ -47,6 +47,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
[RBAC change in external-attacher 3.0](https://github.com/kubernetes-csi/external-attacher/blob/master/CHANGELOG/CHANGELOG-3.0.md#no-really-you-must-read-this-before-you-upgrade)